services/userservice/userservice.go

```diff
@@ -48,3 +49,12 @@ func AuthenticateUserCredentials(emailAddress string, password string) error {
 
 	return nil
 }
+
+func GetUserIdFromEmailAddress(emailAddress string) (models.UserId, error) {
```

Comment:
Hmm... If you make the change I was recommending for the databaseutil function, then this function is basically moot. I'm not sure what makes sense here. I definitely feel weird about these two functions having such different names though. If we keep both of them, I think they should be called `GetIdForUserWithEmailAddress`

Reply:
Personally i'd be a fan or removing `databaseutil.go` and incorporating all sql functions into user service. But i will change them to have the same name, though it can be kinda confusing having two functions whose api look virtually identical but do slightly different stuffs.